arrow-site/docs/docs/core/README.md

```diff
@@ -268,8 +268,8 @@ configuration:
 ```xml
 
 <repository>
-    <snapshotss>
-        <url>https://oss.sonatype.org/content/repositories/snapshots/</url>
+    <url>https://oss.sonatype.org/content/repositories/snapshots/</url>
+    <snapshots>
         <updatePolicy>always</updatePolicy>
     </snapshots>
```

Comment:
```suggestion
```
We are no longer producing snapshots so this piece of config should not be required.